frontend/packages/dapple/package-post-init.js

```diff
@@ -47,13 +45,14 @@ const tokenSpecs = {
   VSL: { precision: 18, format: '0,0.00[0000000000000000]' },
   PLU: { precision: 18, format: '0,0.00[0000000000000000]' },
   MLN: { precision: 18, format: '0,0.00[0000000000000000]' },
+  RHOC: { precision: 9, format: '0,0.00[0000000]'  }
```

Comment:
Same said in the other comment. The total supply is in TOKEN. 1 TOKEN = 1 * 10 ^8 for this case.